pkg/controller/certificates/internal/secretsmanager/keystore.go

```diff
@@ -78,6 +79,20 @@ func encodePKCS12Keystore(password string, rawKey []byte, certPem []byte, caPem
 	return keystoreData, nil
 }
 
+func encodePKCS12Truststore(password string, caPem []byte) ([]byte, error) {
+	ca, err := pki.DecodeX509CertificateBytes(caPem)
+	if err != nil {
+		return nil, err
+	}
+
+	var cas = []*x509.Certificate{ca}
+	truststoreData, err := pkcs12.EncodeTrustStore(rand.Reader, cas, password)
```

Comment:
can be simplified into `return pkcs12.EncodeTrustStore(rand.Reader, cas, password)`

Reply:
Thanks for the review, I will make the adjustment.